### PR TITLE
Implement MCG load balancer incoming subnet rules test

### PR DESCRIPTION
**Pull request by MeridianExplorer**
Also refactored some CloudManager code and its usages for better clarity

### CODE REVIEW

**Review by AviadP — APPROVED** [review]

File: ocs_ci/ocs/constants.py [hunk 1]
```diff
@@ -1296,6 +1296,12 @@
 JAVAS3_POD_NAME = "java-s3"
 AWSCLI_SERVICE_CA_CONFIGMAP_NAME = "awscli-service-ca"
 AWSCLI_TEST_OBJ_DIR = "/test_objects/"
+TEST_NET_1_BLOCK = "192.0.2.0/24"
+TEST_NET_2_BLOCK = "198.51.100.0/24"
+TEST_NET_3_BLOCK = "203.0.113.0/24"
+TEST_NET_MCAST_BLOCK = "233.252.0.0/24"
```
Comment (ebenahar) [inline ocs_ci/ocs/constants.py:1302]: are these subnets applicable to AWS only? If so, maybe better to change the var name to reflect that

**Review by ebenahar — APPROVED** [review]

**Review by keesturam — APPROVED** [review]

**Comment by ebenahar** [comment]
@MeridianExplorer is it verified?

**Comment by ebenahar** [comment]
@parikshithb please take a look

**Review by parikshithb — APPROVED** [review]
lgtm

**Review by PrasadDesala — APPROVED** [review]

**Comment by petr-balogh** [comment]
@PrasadDesala @keesturam @sagihirshfeld  @suchita-g  Was there done some testing?

I see there is another PR:
https://github.com/red-hat-storage/ocs-ci/pull/7421

And maybe it's fixing the issue I see in 4.13 production runs where a lot of mcg tests were skipped.
https://url.corp.redhat.com/a712840

With error like:
```
2023-04-04 17:47:08  12:17:07 - MainThread - /home/jenkins/workspace/qe-deploy-ocs-cluster/ocs-ci/ocs_ci/ocs/resources/cloud_manager.py - INFO  - Creating Underlying Storage aws-uls-d92b7b626ad746589b588af65e804913 in None
2023-04-04 17:47:08  12:17:07 - MainThread - ocs_ci.ocs.resources.cloud_uls - WARNING  - aws_client was initialized as None.
2023-04-04 17:47:08  Please verify the needed credentials were set in auth.yaml, or refer to the OCS-CI documentation -
2023-04-04 17:47:08  ( https://ocs-ci.readthedocs.io/en/latest/usecases/running_tests_on_cluster_not_deployed_by_ocsci.html#access-to-cloud-object-storage-mcg-only )
```

**Pull request merged**